CSS/CSS.sublime-syntax

```diff
@@ -112,6 +112,12 @@ contexts:
       scope: constant.other.color.rgb-value.css
       captures:
         1: punctuation.definition.constant.css
+      # RGBA Hexadecimal Colors
+      # https://en.wikipedia.org/wiki/RGBA_color_space#RGBA_hexadecimal_.28word-order.29
+    - match: '(#)(\h{8})\b'
```

Comment:
It is really a nitpick, but the second capture group is unused and not necessary.